src/content/docs/fr/recipes/modified-time.mdx

```diff
@@ -121,7 +121,7 @@ Apprenez à construire un [plugin remark](https://github.com/remarkjs/remark) qu
    </html>
    ```
 
-   Si vous utilisez la [mise en page Markdown](/fr/guides/markdown-content/#pages-markdown-et-mdx), utilisez la propriété du frontmatter `lastModified` de `Astro.props` dans votre modèle de mise en page.
+   Si vous utilisez la [mise en page Markdown](/fr/basics/layouts/#pages-markdown), utilisez la propriété du frontmatter `lastModified` de `Astro.props` dans votre modèle de mise en page.
```

Comment:
I think this should be:
```suggestion
   Si vous utilisez la [mise en page Markdown](/fr/basics/layouts/#mises-en-page-markdown), utilisez la propriété du frontmatter `lastModified` de `Astro.props` dans votre modèle de mise en page.
```
(see [the file updated in another PR](https://github.com/withastro/docs/blob/3b67e3738225ec96ec5c89f8d882632d7e491a19/src/content/docs/fr/basics/layouts.mdx?plain=1#L99))